pages/Home.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { useEffect } from "react"
+import BundlesOverview from "../components/bundles/BundlesOverview"
+import * as ga from "../lib/ga"
+
+export default function Home() {
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  useEffect(ga.pageview)
```

Comment:
this is also present in `index.tsx`. I'm about confused about the need for this `Home.tsx` file

Reply:
I'm not sure of the intended usage here, there wasn't a higher order component above all the page routes, so assumed that this would need to be on each page